apps/andelabs.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+module.exports = {
+	dataTypes: function(argument){
+		switch (typeof (argument)){
+			case 'number':
+				return getNumberResult(argument); break;
+			case 'string':
+				return argument.length; break;
+			case 'object':
+				if (argument === null){
+					return 'no value'; break;
+				}
+				else{
+					return argument[2]; break;
+				}
```

Comment:
This doesn't cover the edge case of if an actual object is passed in as the argument.